packages/tags/README.md

```diff
@@ -83,3 +89,9 @@ import {
     </sp-tag>
 </sp-tags>
 ```
+
+### Accessibility
+
+`<sp-tags>` is a `role="list"` container that manages a collection of `Tag` elements. It has an `aria-label` attribute that defaults to "Tags".
```

Comment:
Just to sort of follow Rajdeeps suggestions, what do you think about changing "`Tag` elements" to `"`<sp-tag>` elements?